### DISCUSSION TITLE
Bulk import

### DISCUSSION THREAD

**Discussion by abousarsar** [khoait/DCE.PCF#211] [General] [answered]
Hello,
Your work on this control is greatly appreciated, We’ve noticed that the bulk import (from Excel) functionality does not seem to work with the control. 
Could you kindly confirm if this is a known limitation of the component? We are using the N-N standard relationship...
If so, would you have any suggestions or guidance on how we could adapt or improve the control to handle bulk import ? Perhaps using a custom n-n relationship or other way.
I’d be grateful for any insights you can share to help us address this challenge.
Thank you

**Comment by khoait** [comment, marked as answer]
Hi @abousarsar thanks for your comments. The control is only displaying the related data at the front end level, it can't process any changes/events happen in the backend such as importing.
in your case, make sure you import data for table1 and table2, as well as the connection between them stored in the NN table created by the system. You can find the NN table name in the relationship details.
with data import properly, the control will display as expected.
hope this helps!
